inference-engine/src/mkldnn_plugin/mkldnn_plugin.cpp

```diff
@@ -116,12 +127,19 @@ Engine::LoadExeNetworkImpl(const InferenceEngine::ICNNNetwork &network, const st
     std::shared_ptr<ICNNNetwork> clonedNetwork = cloneNetwork(network);
     if (clonedNetwork->getFunction()) {
         Transformation(clonedNetwork);
+        NetPass::ConvertPrecision(*clonedNetwork, Precision::BOOL, Precision::U8);
```

Comment:
why this `ConvertPrecision` is out of `Transform` function?

Reply:
I suppose `QueryNetwork` should have the same function call..

Reply:
BOOL to U8 conversoin in not yet implemented in ngrpah pass because it requires TypeRelaxed class and new RTTI from @slyalin

Reply:
> BTW, how is it supposed to re-use in other plugins? via another PR or plugin developers must do it by ourselves?

I'll update other plugins in a separate PR.

Reply:
NetPass call moved to Transformation function.